randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1234,8 +1234,8 @@ Extra - room_id: [49, 59]
       All of the following:
           # Charge shinespark from Auxilary Power Station https://discord.com/channels/119665192110915584/1199873303892992020/1207854781406777405
           High Jump and Speed Booster and Wave Beam and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Enemy Rando
-          # TODO: tougher without charge
-          Charge Beam
+          # Without charge beam, the shot to open the gate must be precisely timed.  With charge beam, just let go of beam while in the transition.
```

Comment:
Can you Capitalize `Charge Beam` please? Nicer to read.

Reply:
Fixed